engine/Cargo.lock

```diff
@@ -913,7 +913,7 @@ dependencies = [
 
 [[package]]
 name = "baml-python-ffi"
-version = "0.1.0"
+version = "0.78.0"
```

Comment:
Possible to pull this version from the `engine` workspace so it stays in sync?

Reply:
i already did :) checkout the update to cargo.toml